tools/mirror-csv-release/mirror_csv_release.sh

```diff
@@ -164,6 +174,14 @@ main() {
                 bundle_registry_tag="$2"
                 shift 2
                 ;;
+            --bundle-registry-dest)
+                bundle_registry_dest="$2"
+                shift 2
+                ;;
+            --no-mirror)
+                no_mirror="$2"
```

Comment:
I think it would be better to use just an optional flag without expecting any value as we are doing for instance for --dry-run

Reply:
+1 sorry not that skilled in bash that makes a ton more sense!